src/debugtopus.ts

```diff
@@ -76,11 +76,15 @@ export const prepareTestRun = async ({
     packageRootDir = getPackageRootLevel(appDir);
   }
 
-  const tempDir = path.join(packageRootDir, "temp");
+  const tempDir = path.join(packageRootDir, `temp`);
 
   if (!existsSync(tempDir)) {
     await fs.mkdir(tempDir);
+  } else {
+    await fs.rm(tempDir, { force: true, recursive: true });
+    await fs.mkdir(tempDir);
```

Comment:
if we are creating the tempDir in both cases, then reverse the if check and put delete there?

Reply:
true, good point! fixed